keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/nuts/NUTS.java

```diff
@@ -151,57 +154,142 @@ public Statistics getStatistics() {
 
     public static class NUTSBuilder {
         private KeanuRandom random = KeanuRandom.getDefaultRandom();
-        private int adaptCount = DEFAULT_ADAPT_COUNT;
-        private double targetAcceptanceProb = DEFAULT_TARGET_ACCEPTANCE_PROB;
-        private boolean adaptEnabled = true;
+
+        private int adaptCount = 1000;
```

Comment:
I didn't like the DEFAULT_ pattern because it adds a layer of indirection. I've added a comment above theses defaults saying that they are the defaults. We did the DEFAULT_ thing before because it was clearer with the Lombok magic, but we don't use the Lombok builder here anymore.